jinahub/indexers/searcher/compound/FaissPostgresSearcher/tests/unit/test_unit.py

```diff
@@ -0,0 +1,47 @@
+import os
+import time
+
+import numpy as np
+import pytest
+from jina import Document, Flow
+
+from ... import FaissPostgresSearcher
+
+
+@pytest.fixture()
+def docker_compose(request):
+    os.system(
+        f"docker-compose -f {request.param} --project-directory . up  --build -d "
+        f"--remove-orphans"
+    )
+    time.sleep(5)
+    yield
+    os.system(
+        f"docker-compose -f {request.param} --project-directory . down "
+        f"--remove-orphans"
+    )
+
+
+d_embedding = np.array([1, 1, 1, 1, 1, 1, 1])
+c_embedding = np.array([2, 2, 2, 2, 2, 2, 2])
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+compose_yml = os.path.join(cur_dir, '..', 'docker-compose.yml')
+
+
+@pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
+def test_shards_str(docker_compose):
```

Comment:
I would argue, anything that uses a docker-compose is not a unit test anymore and thus have not the renaming to `unit/test_unit.py`.

Reply:
it depends on docker for the PSQL db. Can't really test PSQL without a database. I'll move it to the integration test folder